src/components/FieldWrapper/FieldWrapper.tsx

```diff
@@ -80,7 +86,7 @@ export const FieldWrapper = React.forwardRef<HTMLDivElement, Props>(
                         title={clearTooltipLabel}
                         {...elem('clearButton', { rightIndent: showArrow })}
                     >
-                        <MdClear size={16} />
+                        <MdClear size={16} role="img" aria-label={clearTooltipLabel} />
```

Comment:
What is the profit of using role="img" for SVG inside button? The button tag itself can be read by the reader. We only need to add a label to the button.

Reply:
I think it is good to add clear roles, it allows us to grab elements based on role during testing. I'm not 100% sure of all the benefits, but at least RTL recommends to use that selection method above all others. But indeed, if we do have the label, we can also select by that.

Reply:
I mean why do we need to add some role to the SVG icon? The button is a clickable element, not an icon, right?

Reply:
the role `img` reflects that it is not clickable. It might not be 100% needed, but I also don't see any harm in adding it, and being more explicit.